tools/docker/boardfarm-ci/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.8-slim-buster
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates curl \
+    curl \
+    gcc \
+    git \
+    gnupg \
+    gnupg-agent \
+    libsnmp-dev \
+    software-properties-common \
+    wireshark-common \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /app/requirements.txt
+
+WORKDIR app
+RUN pip3 install -r requirements.txt
```

Comment:
Why do we need this? Before, we just took the requirements.txt from boardfarm. I.e.
```

RUN git clone https://github.com/mattsm/boardfarm.git \
    && cd boardfarm \
    && git checkout 100521fde1fb67536682cafecc2f91a6e2e8a6f8 \
    && pip3 install -r requirements.txt
    && python3 setup.py install 
```

Yes, it does mean that all the pip stuff has to be re-downloaded whenever we change the boardfarm hash, but that's going to be rare anyway.

Reply:
Boardfarm itself is not installed in this setup, it is used from the rootdir/boardfarm dir directly to allow for any changes made locally to be used.

As we do not expect to change boardfarm itself, this will be changed to installing as before, cloning and the rest.

Reply:
@arnout good point, I hadn't noticed boardfarm itself also included the requirements, I though it was added by us.

@odkq with these changes, boardfarm _is_ installed in the image itself. Is it something you want to do differently?